package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xometry/intacct-sdk",
-  "version": "2.2.1",
+  "version": "1.0.0",
```

Comment:
Any reason this is starting before the last original intacct version? Will this impact maintaining parity with their package or is this a complete departure?

Reply:
Ah thats a good call. It probably makes more sense to maintain the original and then just branch from there